chanzuckerberg/idseq-web
# [Maps] improve perf metadata by sample

# Description

Improve performance of the `metadata_multiget` method by eager loading (v2) locations. 

# Tests

* Make a request for samples (the domain or specified samples should have regular metadata and location v2. e.g. `http://localhost:3000/samples/index_v2.json?domain=my_data&sampleIds=<sampleId>`
* Verify that metadata field loads correctly, include extra object for location v2.
